src/workerd/server/server.c++

```diff
@@ -1243,13 +1243,51 @@ public:
 
     const ActorConfig& getConfig() { return config; }
 
-    kj::Own<IoChannelFactory::ActorChannel> getActor(Worker::Actor::Id id) {
+    kj::Own<WorkerInterface> getActor(Worker::Actor::Id id,
+        IoChannelFactory::SubrequestMetadata metadata) {
+      auto promise = getActorImpl(kj::mv(id))
+          .then([this, metadata = kj::mv(metadata)](kj::Own<Worker::Actor> actor) mutable {
+        return service.startRequest(kj::mv(metadata), className, kj::mv(actor));
+      });
+
+      return newPromisedWorkerInterface(service.waitUntilTasks, kj::mv(promise));
+    }
+
+    kj::Own<IoChannelFactory::ActorChannel> getActorChannel(Worker::Actor::Id id) {
+      return kj::heap<ActorChannelImpl>(*this, kj::mv(id));
+    }
+
+  private:
+    WorkerService& service;
+    kj::StringPtr className;
+    const ActorConfig& config;
+    kj::HashMap<kj::String, kj::Own<Worker::Actor>> actors;
+
+    class Loopback : public Worker::Actor::Loopback, public kj::Refcounted {
```

Comment:
Nit: can you add a doc comment in here briefly explaining what the purpose of `Loopback` is